src/runtime/hexagon/hexagon_buffer_manager.h

```diff
@@ -85,6 +86,18 @@ class HexagonBufferManager {
     return hexagon_buffer_map_.empty();
   }
 
+  //! \brief Returns a list of allocated pointers.
+  // Note - this should only be used by the device API to keep track of what
+  // was in the manager when HexagonDeviceAPI::ReleaseResources is called.
+  std::vector<void*> vector() {
```

Comment:
Nit: Renaming to `current_allocations` would tell a reader what is contained in the return value.